pkg/goods/support/host-support-bundle.tmpl.yaml

```diff
@@ -156,9 +284,39 @@ spec:
       collectorName: 'check-umount'
       command: 'sh'
       args: ['-c', 'command -v umount']
+  - run:
+      collectorName: "mount"
+      command: "mount"
+      args: ["-l"]
   - copy:
       collectorName: installer/lam-service-config
       path: /etc/systemd/system/local-artifact-mirror.service.d/*
+  - run:
+      collectorName: "ps-high-load"
+      command: "sh"
+      args: ["-c", "ps -eo s,user,cmd | grep ^[RD] | sort | uniq -c | sort -nbr | head -20"]
+  - run:
+      collectorName: "ps-detect-antivirus-and-security-tools"
+      command: "sh"
+      args: [-c, "ps -ef | grep -E 'clamav|sophos|esets_daemon|fsav|symantec|mfend|ds_agent|kav|bdagent|s1agent|falcon|illumio|xagt' | grep -v grep"]
+  - filesystemPerformance:
+      collectorName: filesystem-latency-two-minute-benchmark
+      timeout: 2m
+      directory: /var/lib/etcd
+      fileSize: 22Mi
+      operationSizeBytes: 2300
+      datasync: true
+      enableBackgroundIOPS: true
+      backgroundIOPSWarmupSeconds: 10
+      backgroundWriteIOPS: 300
+      backgroundWriteIOPSJobs: 6
+      backgroundReadIOPS: 50
+      backgroundReadIOPSJobs: 1
+      exclude: true
```

Comment:
This isn't the right folder for sure, we should also check if this is the most recent test. I think EC already tests filesystem performance now. Is this the same as the Preflight?

At the very least it's not a two-minute-benchamark so I'm pretty sure you've lifted the old one.

Reply:
yeah this is a lift from the old one - will update the data directory, good catch

Reply:
I believe this is the new one: https://github.com/replicatedhq/embedded-cluster/blob/cf9f38c56e978ee704e6d9b5e21c1dd9b564d88a/pkg/preflights/host-preflight.yaml#L429

Reply:
done